src/main/java/org/apache/maven/buildcache/CacheResult.java

```diff
@@ -70,7 +70,7 @@ public static CacheResult failure(CacheContext context) {
         return new CacheResult(RestoreStatus.FAILURE, null, context);
     }
 
-    public static CacheResult rebuilded(CacheResult orig, Build build) {
+    public static CacheResult rebuilt(CacheResult orig, Build build) {
```

Comment:
This is an API breaking change, so keep the old method as a delegate to the new method and deprecate it.

Reply:
Fixed. Try now.